### ISSUE TITLE
Adversarial Robustness of trained models does not match results in the paper

### ISSUE THREAD

**Issue by pratyushmaini** [ddkang/advex-uar#6]
Hi, I am testing the pre-trained model (l_inf,8/255) using PGD attacks (10 steps, alpha = 1.6/255) and the robust accuracy is less than 30% as opposed to that mentioned in the paper as being > 50%. Can you confirm if you have seen this observation before since the paper was published? If not, I would be happy to share more details.

**Comment by yi-sun** [comment]
I'm assuming this is on ImageNet-100 and alpha is the step size?  The number reported in the paper is 59 for the (l_inf, 8/255) adversarially trained model against the (l_inf, 8/255, 200 steps, step size 0.57) attack.  Can you specify how to reproduce what you are seeing?
